pyramid/eviction.go

```diff
@@ -59,5 +59,7 @@ func (re *ristrettoEviction) store(rPath relativePath, filesize int64) bool {
 }
 
 func (re *ristrettoEviction) onEvict(item *ristretto.Item) {
-	re.evictCallback(item.Value.(relativePath), item.Cost)
+	if item.Value != nil {
```

Comment:
`ristrettoEviction` is coupled with the implementation of `TierFS`. I would much rather check for nil `onEvict` callback during creation, which isn't really allowed, than checking for nil here.